src/lang/en-US.json

```diff
@@ -1,7 +1,6 @@
 {
   "yes": "yes",
   "no": "no",
-
```

Comment:
i'm not sure if the script can preserve whitespace but it would be nice to keep the separation lines

Reply:
If I read the file line by line then it could.  I'm just reading the file in as JSON, doing the replace, and then writing it out (using JSON.stringify(obj, undefined, "  ") to preserve some whitespace.

Reply:
I think you could just do a replace on the raw text, I don't think %s and %d ever show up in the keys or delimiters.